addons/account_peppol/models/res_partner.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import requests
+from lxml import etree
+from hashlib import md5
+from urllib import parse
+
+from odoo import api, fields, models
+
+TIMEOUT = 30
```

Comment:
Are the requests that slow? Isn't 5 seconds enough?
```suggestion
TIMEOUT = 5
```

Reply:
I'm not sure, I took the same timeout as we have on the iap side